### PR TITLE
Update strings.xml bg

### PR DESCRIPTION
**Pull request by REMPO6**
my id/29369269

### CODE REVIEW

File: res/values-bg/strings.xml [hunk 1]
```diff
@@ -1305,14 +1305,14 @@ Commands:
...
+    <string name="_2x_skin_resolution">2x Резолюция на кожата</string>
     <string name="pm">PM</string>
-    <string name="You_are_already_friends_">You are already friends.</string>
-    <string name="Account_must_be_1_day_old_to_send_mail_">Account must be 1 day old to send mail.</string>
-    <string name="Translator">Translator</string>
-    <string name="SUCCESS">SUCCESS</string>
+    <string name="You_are_already_friends_">Вече сте приятели.</string>
+    <string name="Account_must_be_1_day_old_to_send_mail_">Акаунтът трябва да е на един ден, за да изпратите имейл.</string>
+    <string name="Translator">преводач</string>
+    <string name="SUCCESS">успех</string>
```
Comment (Jeii20) [inline res/values-bg/strings.xml:1315]: Line 1314: First letter must be capital
Line 1315: Please keep this 100% capital letters

**Review by Jeii20 — CHANGES_REQUESTED** [review]

File: res/values-bg/strings.xml (hunk 1 above)
Comment (REMPO6) [reply res/values-bg/strings.xml]: They have been modified.

**Review by Jeii20 — APPROVED** [review]
Thanks for your time ^^
Please mark the conversation as resolved when you can

File: res/values-bg/strings.xml [hunk 2]
```diff
@@ -1305,14 +1305,14 @@ Commands:
     <string name="language_change_warning">Changing the language will cause the app to restart.</string>
     <string name="xmas_gift_boost_message">You\'ve received a gift of 3x plasma boost for 1 day!</string>
     <string name="no_response">No response from server. Please check the internet connection.</string>
-    <string name="RAMADAN">RAMADAN</string>
-    <string name="SUDDEN_ATTACK">SUDDEN ATTACK</string>
-    <string name="_2x_skin_resolution">2x Skin Resolution</string>
+    <string name="RAMADAN">Рамадан</string>
```
Comment (Jeii20) [inline res/values-bg/strings.xml:1308]: Please keep here 100% capital letters

File: res/values-bg/strings.xml [hunk 3]
```diff
@@ -1305,14 +1305,14 @@ Commands:
...
+    <string name="SUDDEN_ATTACK">ВНЕЗАПНА АТАКА</string>
+    <string name="_2x_skin_resolution">2x Резолюция на кожата</string>
```
Comment (Jeii20) [inline res/values-bg/strings.xml:1310]: "К" in "кожата" must be capital

**Comment by Jeii20** [comment]
PR closed. Same PR with same changes + corrections identified at #472.

**Pull request closed without merging**